src/main/java/servlets/SellerLoginServlet.java

```diff
@@ -16,7 +16,8 @@
 import com.bittercode.service.UserService;
 import com.bittercode.service.impl.UserServiceImpl;
 
-public class SellerLoginServlet extends HttpServlet {
+//Suppressed warning for same reason in CustomerLoginServlet.java file

```

Comment:
Comments like this are generally a bad idea, for two reasons:
* they make the reader do extra work to find the other file
* they're likely to become out-of-date: what if that other file is changed so that the reasons are no longer the same? There's no way for a maintenance programmer to know that they need to change this comment, too.